zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -42,6 +42,8 @@ pub struct Block<F> {
     pub copy_events: Vec<CopyEvent>,
     /// Length to rw table rows in state circuit
     pub state_circuit_pad_to: usize,
+    /// Inputs to the SHA3 opcode
+    pub sha3_inputs: Vec<Vec<u8>>,
```

Comment:
Maybe is clearer to use:
```suggestion
    pub sha3_inputs: Vec<Bytes>,
```

Reply:
I'mt no a big fan of the `Bytes` type because:
- I think `Vec<u8>` is more clear as it just uses std types
- The `Vec` type has many useful methods that we can access directly.  And we avoid having to do things like `to_vec()`, or `as_ref()` everywhere
- The `Bytes` type main purpose is for json (serial/deserial)ization as a hex string, and we don't need that here: https://docs.rs/ethers/latest/ethers/types/struct.Bytes.html

So unless you feel very strong about it, I'd prefer keeping it like it is :)

Reply:
No strong feelings! So feel free to go this way :smile: